test/integration-tests/NFTX.test.ts

```diff
@@ -25,82 +18,78 @@ describe('NFTX', () => {
   let alice: SignerWithAddress
   let router: UniversalRouter
   let permit2: Permit2
-  let cryptoCovens: ERC721
+  let miladyContract: ERC721
   let twerkyContract: ERC1155
   let planner: RoutePlanner
 
   beforeEach(async () => {
-    await resetFork()
+    await resetFork(17029001) // 17029002 - 1
     await hre.network.provider.request({
       method: 'hardhat_impersonateAccount',
       params: [ALICE_ADDRESS],
     })
     alice = await ethers.getSigner(ALICE_ADDRESS)
-    cryptoCovens = COVEN_721.connect(alice)
+    miladyContract = MILADY_721.connect(alice)
     twerkyContract = TWERKY_1155.connect(alice)
     permit2 = (await deployPermit2()).connect(alice) as Permit2
     router = (await deployUniversalRouter(permit2)).connect(alice) as UniversalRouter
     planner = new RoutePlanner()
   })
 
   it('completes an ERC-721 buyAndRedeem order with random selection', async () => {
-    const value = expandTo18DecimalsBN(4)
-    const numCovens = 2
+    const value = expandTo18DecimalsBN(2.036523961400441269)
+    const numMiladys = 1
     const calldata = nftxZapInterface.encodeFunctionData('buyAndRedeem', [
-      NFTX_COVEN_VAULT_ID,
-      numCovens,
+      NFTX_MILADY_VAULT_ID,
+      numMiladys,
       [],
-      [WETH.address, NFTX_COVEN_VAULT],
+      '0xd9627aa400000000000000000000000000000000000000000000000000000000000000800000000000000000000000000000000000000000000000001bfb8d0ff32c43470000000000000000000000000000000000000000000000000e27c49886e6000000000000000000000000000000000000000000000000000000000000000000010000000000000000000000000000000000000000000000000000000000000002000000000000000000000000c02aaa39b223fe8d0a0e5c4f27ead9083c756cc2000000000000000000000000227c7df69d3ed1ae7574a1a7685fded90292eb48869584cd00000000000000000000000010000000000000000000000000000000000000110000000000000000000000000000000000000000000000465b3a7f1b643618cb',
```

Comment:
What is this..? Looks like it includes the WETH address and some other stuff?

Reply:
In the new Zap contract instead of passing in the path to `buyAndRedeem`, we pass in swapData, which I assume encodes the router + swap path used